package.json

```diff
@@ -7,16 +7,25 @@
     "prettier-hook": "pretty-quick",
     "prettier-check-hook": "pretty-quick --check",
     "prepublishOnly": "make move-package-json-to-dist",
-    "update-version": "make update-entry"
+    "update-version": "make update-entry",
+    "test": "jest --env=jsdom"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.23.3",
```

Comment:
what I don't like is that we can't run tests.. it's a big change adding babel and all libs
Does babel addition affect the build somehow?

Reply:
this is only on the global level
We have our react SDK inside `react` folder that has its own build tools and rules